ioctl/cmd/action/actionhash.go

```diff
@@ -167,10 +167,12 @@ func printActionProto(action *iotextypes.Action) (string, error) {
 	if err != nil {
 		return "", output.NewError(output.ConvertError, "failed to convert bytes into address", err)
 	}
+	//ioctl action should display IOTX unit instead Raul
+	gasPriceUnitIOTX, err := util.StringToIOTX(action.Core.GasPrice)
```

Comment:
check err
or use gasPriceUnitIOTX, _ := if we are sure action.Core.GasPrice is correct